modules/local/add_found_in_tag/main.nf

```diff
@@ -76,12 +76,12 @@ process ADD_FOUND_IN_TAG {
                     args2.contains("--output-type z") || args2.contains("-Oz") ? "vcf.gz" :
                     args2.contains("--output-type v") || args2.contains("-Ov") ? "vcf" :
                     "vcf"
-    def index = args2.contains("--write-index=tbi") || args2.contains("-W=tbi") ? "tbi" :
+    def index_ = args2.contains("--write-index=tbi") || args2.contains("-W=tbi") ? "tbi" :
```

Comment:
Here I would use a leading underscore rather than trailing. See later comment
```suggestion
    def _index = args2.contains("--write-index=tbi") || args2.contains("-W=tbi") ? "tbi" :
```